tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -783,7 +783,6 @@ type ProjectWorkflowBuilder
         initialProject: SyntheticProject,
         ?initialContext,
         ?checker: FSharpChecker,
-        ?useGetSource,
```

Comment:
Not sure if this won't break some tests.

If we make this change we should probably rename `useChangeNotifications` to something more fitting, like `useInMemoryDocuments` or `useFileSystem` (this would be the inverse condition).